src/server/api/medias/index.js

```diff
@@ -293,11 +296,19 @@ router.get('/url', (req, res) => {
       return res.status(400).send(new Error('Incorrect format for quarter'));
     }
 
+    if (BRUINCAST_HASH_CLIENT_IP === 'true') {
```

Comment:
@jackvanb So boolean values in .env are converted to strings?

Reply:
All .env values are treated as strings. (https://github.com/motdotla/dotenv/issues/51)

This seems to be the clearest way to express the boolean logic.